File: faas/sign-image/handler.py
from cloudinary import utils
from json import dumps
from os import getenv
from time import time


def handle(req):
    timestamp = int(time())
    signature = utils.api_sign_request(
        {"timestamp": timestamp}, getenv("CLOUD_SECRET"))

    result = {
        "signature": signature,
        "timestamp": timestamp
    }

    return dumps(result)

File: faas/sign-image/handler.js
"use strict";
const cloudinary = require("cloudinary");

module.exports = (event, context) => {
    const timestamp = Math.round(new Date().getTime() / 1000);
    const signature = cloudinary.utils.api_sign_request(
        { timestamp: timestamp },
        process.env.CLOUD_SECRET
    );
    const payload = {
        signature: signature,
        timestamp: timestamp
    };

    context.status(200).succeed(payload);
};
